src/lang/en.json

```diff
@@ -118,6 +119,7 @@
         "SP": "BC Sole Proprietorship",
         "GP": "BC General Partnership",
         "ULC": "BC Unlimited Liability Company",
+        "C": "BC Limited Company",
```

Comment:
Is there a ticket to add other legalTypes ? Is there details in that ticket that confirm these are all that we can see in Business Dash?